app/views/devise/sessions/new.html.erb

```diff
@@ -19,8 +36,16 @@
   <% end %>
 
   <div class="actions">
-    <%= f.submit "Log in" %>
+    <%= f.submit "Log in"  , class: 'submit-btn'%>
   </div>
-<% end %>
+<%end%>
 
 <%= render "devise/shared/links" %>
```

Comment:
Error message currently comes from "devise/shared/links", which makes it difficult access with CSS and style it. You can consider creating the error message yourself.